sympy/utilities/runtests.py

```diff
@@ -242,6 +250,10 @@ def run_all_tests(test_args=(), test_kwargs={}, doctest_args=(),
     """
     tests_successful = True
 
+    test_kwargs = test_kwargs or {}
```

Comment:
No. There isn't any error checking so you might as well just assign a dictionary if a null value (e.g. {} or None is sent). But maybe I am missing something that you are seeing. Is there a reason that you were explicit there but not here? If not, I would prefer to see the same `test_kwargs = test_kwargs or {}` replacing lines 205-206.